rdm/0006-profiles-and-flavours.md

```diff
@@ -0,0 +1,104 @@
+- Start Date: 2019-10-02
+- RFC PR: [#6](https://github.com/inveniosoftware/rfcs/pull/6)
+- Authors: Guillaume Viger
+
+# Customize created Invenio instance by choosing profile and optionally flavour
+
+## Summary
+
+When running the instance bootstrapping CLI command (`invenio-cli init`), the
+hoster should be given a choice to create an Integrated Library System (ILS) or
+Resource Data Management (RDM) "flavour" Invenio application. If RDM is chosen,
+there are further "profiles" to be chosen from, for instance "default" and
+"biomed". These choices pre-configure the application to varying levels.
+
+## Motivation
+
+> Why are we doing this? What user stories does it support? What is the expected
+outcome?
+
+As a (prospecting-)hoster, I want to have a quick working application for my
+main use case: library system or data management, so I can evaluate it
+right away (prospecting) and/or start using it (configuration/development). The
+RDM choice implies different kinds of data management for which I want to leverage
+pre-configuration / pre-customizations in order to get started faster and adopt
+same settings as other installations in my domain.
+
+As an InvenioRDM (core-, extension-, plugin-)developer for domain stakeholder,
+I want to be able to satisfy the exclusive needs/wants of my specialized
+stakeholders while still being able to develop and use domain authorities **within**
+the InvenioRDM context. I have a dedicated mechanism and space to insert
+customizations shared by instances for my domain this way.
+
+
+## Terminology
+
+**flavour**: A *flavour* represents the main use case of the application
+(e.g., ILS or RDM for now) and it translates to a different packaged application.
+The flavour packaged applications are completely different, which calls
+for a different project repository to define them.
+
+**profiles**: *Profiles* are different domains of interest (e.g., Generic or BioMed)
+for the InvenioRDM *flavour* and translate to different configurations,
+data models, and customizations. Those are not major enough to warrant being
```

Comment:
Major enough is correct. However, I believe this is also more of a hierarchy/abstraction layers. If we think about it all profiles could be flavours, but flavours have a high percentage in common. What do you think? I am not sure how to reword this part.

Reply:
Reworded to get at what you are saying. See if that works better.